projects/vcx.md

```diff
@@ -0,0 +1,97 @@
+* Interoperability: As mentioned, VCX comes from an Aries ecosystem, where interoperability is at the forefront. VCX holds itself accountable to this by participating in efforts such as the OWF OWL test harness.
+* Open Source Development: VCX has always been community driven, and encourages participation with efforts such as fortnightly community calls and previous participation in the Linux Foundation Mentorship programs.
+* Support for Decentralized Identity: VCX's purpose is to provide tools that make it easier for consumers to develop DI applications. VCX has been proven to succeed in this, with testimonies in the industry (such as from [Anonyome Labs](https://anonyome.com/resources/blog/aries-vcx-anonyomes-commitment-to-decentralized-identity/)).
+
+# Code of Conduct
+
+VCX currently follows the Hyperledger Code of Conduct, which is what the OWF code of conduct is based on.
+
+# TAC Sponsor
+
+Stephen Curran
+Second Sponser - TBD
+
+# Project License
+
+Apache 2.0
+
+# Source Control
+
+Project already exists on Github: https://github.com/hyperledger/aries-vcx .
+
+Ideally the new slug would be `vcx`. Dropping the Aries to reflect the capabilities beyond Aries protocols.
+
+# Issue Tracker
+
+https://github.com/hyperledger/aries-vcx/issues
+
+# External Dependencies
+
+The VCX dependency list is maintained in the [source repository](https://github.com/hyperledger/aries-vcx/blob/main/Cargo.toml). All are open source and we believe (but to be verified) that the majority use the Apache 2 License.
+
+TODO - merge workspace dep related PRs for easier dep view
+
+# Release Methodology
+
+All proposed repositories have continuous deployment/delivery pipelines built using GitHub Actions. The individual packages follow the semantic versioning method, ensuring consistency and safety.
+
+VCX publishes release on on an as-needed basis, but releases tend to be made every 4-12 weeks (6 releases were published in 2024).
+
+TBD - VCX does not currently publish any crate releases to crates.io, but would like to in the future. VCX crates are currently "released" via github tags, which Rust consumers consume via github dependencies.
+
+# Initial Maintainers
+
+- George Mulhearn [@gmulhearn](https://github.com/gmulhearn) - Anonyome Labs
+- James Ebert - [@JamesKEbert](https://github.com/JamesKEbert) - Instnt
+- Luke Li - [@lukewli-anonyome](https://github.com/lukewli-anonyome) - Anonyome Labs
+
+# Proposed Project Governance
+
+The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the Aries Project Charter. In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
+
+# Financial Sponsorship
+
+Hyperledger has covered infrastructure related costs. 
```

Comment:
Will Hyperledger keep sponsoring the project?

Reply:
I don't anticipate this, I believe everything would be moved over to the OWF.